main_test.go

```diff
@@ -134,7 +135,7 @@ func setupTest() {
 		if err != nil {
 			panic(err)
 		}
-		if _, err := gz.Write([]byte(`{"foo":bar, "hello:world", "answer":42, "question":null}`)); err != nil {
+		if _, err := gz.Write([]byte(fmt.Sprintf(`{"foo":bar, "hello:world", "answer":42, "question":null, "useragent": %s}`, r.Header.Get("User-Agent")))); err != nil {
```

Comment:
It feels a bit gross for the response to roundtrip the requestor's `User-Agent` in the export data response body (even if it's for the sake of testing). Some possibilities would be to create a new client `client3` with either a custom Transport in which the outgoing request can be inspected or overriding the client's `Do` method.  We can also just skip adding a test, too.

Reply:
Yeah I wasn't sure myself about the value of this test. I'm leaning towards removing it too. Verifying the outgoing request seems silly after explicitly setting the header. I'm fine not including a test, if you don't think its necessary either.

Reply:
Yeah, not adding a test is cool.

Reply:
cool, removed the test. PTAL